packages/stores/src/stores/FeatureFlagsStore.ts

```diff
@@ -2,6 +2,8 @@ import BaseStore from './BaseStore';
 
 const FLAGS = {
     next_wallet: false,
+    trade_sharkfin: false,
+    // Add your flag here 🚀 Note: Trade feature flags should start with 'trade_' prefix.
```

Comment:
```suggestion
```
@maryia-deriv Let's not make custom rules when it's not necessary, Dev should be able to add any flag name which makes sense for their need. Grouping things is not a good idea, For example this `trade_something` might also be used in `@deriv/core` for some reason, So grouping it to `@deriv/trader` doesn't make much sense.

Reply:
@farzin-deriv I had to add this prefix only for a single reason described here https://github.com/binary-com/deriv-app/pull/9816/files#r1329668992 , i.e. to only hide feature flags for DTrader from Endpoint page for app.deriv.com, and not affect the rest of the flags, e.g. for wallets.